docs/intro/welcome.md

```diff
@@ -35,6 +35,14 @@ To learn more about the design system, design principles, contribution flow, or
 </Box>
 ```
 
+## Community Heroes
+
+TDS Community depends on your valuable contributions in order to evolve into a beautiful, robust and efficient design system.
+This makes all of our contributors nothing short of Heroes to us. 
+Check out who our Community Heroes are, and find out how you too can get featured on our [Community Heroes](https://tds.telus.com/community-heroes) page. 
+#### It's more than just bragging rights:
+Telus employees that work at the Toronto or Vancouver offices are eligible to redeem **swag** based on their contributions.
```

Comment:
Should be TELUS instead of Telus.

Reply:
Updated in [5001f42](https://github.com/telus/tds-community/pull/169/commits/5001f423c4495752cdb47b6f43386e68da1598e5)